interview-questions.md

```diff
@@ -8,50 +8,50 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-Your answer:
+Your answer: 
 
-Researched answer:
+Researched answer: Git is a control system made to access 
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: git keeps the logs of your updated and old code on your own computer, while github is git except a cloud where others can use with the updated and old code
 
-Researched answer:
+Researched answer:Git is a control system that manages your source code history, while github is cloud based hosting that manages git repositories and allows you to have open source projects that use git. 
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: logical and && or logical or || which give you true or false
 
-Researched answer:
+Researched answer: && and || and =!
```

Comment:
This answer meets the bare minimum requirements of this section.  Researched answers should provide much more content along with complete sentences to be considered higher than a complete status.